aws_crt_common_ws/aws-crt-common-sys/build.rs

```diff
@@ -0,0 +1,337 @@
+use aws_crt_c_flags::CRTModuleBuildInfo;
+use std::borrow::{Borrow, BorrowMut};
+use std::path::Path;
+
+fn load_compiler_configuration(build_config: &mut CRTModuleBuildInfo) {
+    let mut config_file_str: String = "#ifndef AWS_COMMON_CONFIG_H
+#define AWS_COMMON_CONFIG_H
+
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+/*
+ * This header exposes compiler feature test results determined during cmake
+ * configure time to inline function implementations. The macros defined here
+ * should be considered to be an implementation detail, and can change at any
+ * time.
+ */\n"
+        .to_string();
+
+    build_config.add_public_define("AWS_NO_STATIC_IMPL", "1");
+    config_file_str = format!("{}#define AWS_NO_STATIC_IMPL 1\n", config_file_str);
+
+    #[target_feature(enable = "avx2")]
```

Comment:
This doesn't work in a cross-compile situation. Do we need to worry about that?

Reply:
wait how? target_feature is derived from the target arch of the compile run.